.gitignore

```diff
@@ -368,3 +368,7 @@ MigrationBackup/
 
 # Fody - auto-generated XML schema
 FodyWeavers.xsd
+
+# CMake build outputs
+build/
```

Comment:
Is this separate from the "out/build" paths set in CMakePresets.json?

Reply:
Looks like something I did makes this no longer necessary. I'll remove.